TTS/tts/models/vits.py

```diff
@@ -658,8 +667,13 @@ def test_run(self, ap) -> Tuple[Dict, Dict]:
         test_audios = {}
         test_figures = {}
         test_sentences = self.config.test_sentences
-        aux_inputs = self.get_aux_input()
+        if hasattr(self, "speaker_manager"):
+            aux_inputs = self.speaker_manager.get_random_speaker_aux_input()
+        else:
+            aux_inputs = self.get_aux_input()
+
```

Comment:
To synthesize test sentences during multispeaker model training it is necessary to obtain the speaker_ids or d_vecs of some speaker. From this model I implemented a function in speaker_manager that returns a randomly chosen speaker (as was done in the past before the trainer api)